biojava-structure/src/test/java/org/biojava/nbio/structure/align/ce/CeCPMainTest.java

```diff
@@ -405,16 +407,31 @@ public void testCalculateMinCP() throws SecurityException, NoSuchMethodException
 	@Test
 	public void testCECP1() throws IOException, StructureException{
 
-		String name1 = "PDP:3A2KAc";
-		String name2 = "d1wy5a2";
+		String pdb1 = "3A2K";
+		//String name1 = "PDP:3A2KAc"; // A : 234-333
+		String pdb2 = "1WY5";
+		//String name2 = "d1wy5a2"; // A : 217-311
 
+		AtomCache cache = new AtomCache();
 
-		CeCPMain algorithm = new CeCPMain();
+		// since BioJava 6.0.0, there's no PDP provider. The below corresponds to domain "PDP:3A2KAc"
+		List<ResidueRange> ranges = new ArrayList<>();
+		ranges.add(new ResidueRange("A", new ResidueNumber("A",234, null), new ResidueNumber("A", 333, null)));
+		SubstructureIdentifier ssi = new SubstructureIdentifier(pdb1, ranges);
+		Structure structure1 = cache.getStructure(pdb1);
+		ssi.reduce(structure1);
 
-		AtomCache cache = new AtomCache();
+		// since BioJava 6.0.0, there's no RemoteSCOP provider. The below corresponds to domain "d1wy5a2"
```

Comment:
Does this mean it is no longer possible to pass a SCOP identifiers directly?

Reply:
The BerkeleyScopInstallation should still be working. The RemoteScopInstallation was grabbing all data from source.rcsb.org and there was no other solution than removing it.

For the test I wanted to avoid external resources as much as possible, so I directly hard-coded the residue range. I'm not sure how exactly the Berkeley installation works, but most likely it would have to download a lot of data, which wouldn't have been good for the test.

Reply:
Ok, that is what I imagined but I wasn't sure why you were using hard-coded ranges. Makes sense!

Reply:
Yes, Berkeley provides all data in flat files, so it's a big download.